common/src/main/java/com/wynntils/screens/base/widgets/BasicTexturedButton.java

```diff
@@ -20,23 +20,58 @@ public class BasicTexturedButton extends WynntilsButton {
     private final Consumer<Integer> onClick;
     private List<Component> tooltip;
 
+    private boolean renderTooltipAbove;
```

Comment:
`renderTooltipAbove`, above what, the mouse? I suggest you rename this to `renderTooltipAboveMouse` then

Reply:
I think the intention is if the tooltip should have its top position at the mouse position, or its bottom (which is used if the mouse is too close to the bottom of the screen). I think we have used similar names in other places were we handle this, but we can probably think of something better. `tooltipOverlaidUpwards` perhaps?

Reply:
(and we should probably change the other places as well, then; but that need not be done as part of this PR)

Reply:
Exactly as @magicus said. Though I could not find similar names in other places.
If I'm not mistaken, the default behaviour was to display the tooltip upward because `BasicTexturedButton` were only used for the icon at the bottom of the map :

![image](https://github.com/Wynntils/Artemis/assets/21969306/9270cceb-7b8d-4270-ad11-86115ca478a0)

Now, I want to use it for an icon that is on top of container screens. Because English is clearly not my native language, I'll let you decide what you prefer as a name for this attribute.

I think `renderTooltipAboveMouse` is fine `tooltipOverlaidUpwards` might be better. Either way it's pretty straightforward.

Reply:
Well, okay. This is changing in 1.20, I will fix it there...